### PR TITLE
Feat/017 frontend create admin dashboard

### PR DESCRIPTION
**Pull request by EliasSalom**

### CODE REVIEW

File: libs/system-design/src/lib/Table/Table.tsx
Comment (HassanHassouna) [inline libs/system-design/src/lib/Table/Table.tsx:1]: Please remove unnecessary code!

**Review by HassanHassouna — CHANGES_REQUESTED** [review]
Please handle the comments

File: libs/system-design/src/lib/Table/Table.tsx
Comment (EliasSalom) [reply libs/system-design/src/lib/Table/Table.tsx:1]: I update the comment we will use this code but not now, I already put a `TODO`

**Comment by EliasSalom** [comment]
the comment has been resolved and the code updated, waiting for a new review!

**Review by HassanHassouna — APPROVED** [review]

**Pull request merged**